src/main/java/core/basesyntax/dao/writeReportImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Account;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class writeReportImpl implements writeReport {
```

Comment:
The implemented interface `writeReport` should be renamed to `WriteReport` to follow Java naming conventions.